src/Client.php

```diff
@@ -371,4 +380,22 @@ protected function paramsToQueryString(array $params = [])
         }
         return implode('&', $query_string);
     }
+
+    /**
+     * @return float|int
+     */
+    public function getConnectTimeout(): float
```

Comment:
It looks like in the annotation you've specified `float|int` but in the return type there is only `float`.
In addition, currently, the project is not using built-in PHP types. It might be good to have PHP types but it should be done as a separate PR.